scripts/linters/test_files/invalid_quote.py

```diff
@@ -41,4 +41,4 @@ def fake_method(self, content):
         Returns:
             quote(urllib object): Returns quote object.
         """
-        return urllib.quote(content) # Use of quote() is not allowed.
+        return urllib.parse.quote(content) # Use of quote() is not allowed.
```

Comment:
Please remove this file as we are removing the lint check for `urllib\..*quote\(`

Reply:
@DubeySandeep Done !